performance-tests/index.mjs

```diff
@@ -0,0 +1,141 @@
+// @flow strict
+// import url from 'url';
+import playwright from 'playwright';
+import { createWorker, PSM } from 'tesseract.js';
+import waitForRenderedItems from './waitForRenderedItems.mjs';
+
+const testURL = new URL('http://localhost:8888/integration-test/masonry');
+const DEFAULT_SCROLL_COUNT = 5;
+const PINS_COUNT = 5;
+const PAGE_HEIGHT = 1000;
+
+/**
+ * Args:
+ * --scroll-count: number
+ *  Number of times to scroll the page and fetch items
+ * --debug: flag
+ *  Open browser with dev tools, more verbose logging, and don't close browser when done
+ * --verbosity: 'silent' | 'default' | 'verbose'
```

Comment:
i think this is fine as a utility for testing, but let's split some of this code out so we can reuse it to write an actual integration test.

also, we should likely move this folder under `playwright` instead of creating a new top level dir

Reply:
I moved the file to the playwright directory and changed it to be a normal test.